core/models.py

```diff
@@ -190,8 +190,14 @@ def apply_filters(self, filtering):
         if model_filtering is not None:
             for field_name in model_filtering:
                 value = filtering.get(field_name, None)
-                if value is not None:
-                    qs = qs.filter(**{field_name: value})
+                if value == 'false':
+                    value = False
+                elif value == 'true':
+                    value = True
+                elif value is None:
+                    value = True
+                    field_name += '__isnull'
```

Comment:
Acho que esse último `elif` tá errado. Se eu não passar nada no campo `x` mas passaar em `y` (exemplo: `url?x=&y=test`), o código *não deveria*  filtrar por `x__isnull=True` (nesse caso ignoramos o `x`, pois apesar de aparecer na QS ele tá vazio). Só deveríamos filtrar por `x__isnull=True` caso venha `None` na QS (que é o que o formulário de filtro tá fazendo hoje), ou seja: `url?x=None&y=test`.

Reply:
Hmmm ok, não me liguei que tinha o formulário de filtro. Então vou mover essa lógica toda pra ele pra facilitar mesmo.

Reply:
Talvez não valha a pena mover isso pro form agora, porque se não me engano essa função é usada pela API mesmo (tá ruim, mas acho que só vai dar pra desacoplar bem depois que esas parte dos modelos dinâmicos for desacoplada).